simplecov.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency "json", ">= 1.8", "< 3"
   gem.add_dependency "simplecov-html", "~> 0.10.0"
-  gem.add_dependency "docile", "~> 1.1.0"
+  gem.add_dependency "docile", [">= 1.1.0", "< 1.3"]
```

Comment:
@jaredbeck @PragTob Is this dependency declaration still unnecessarily restrictive?

Respecting semantic versioning, shouldn't it be sufficient to say:

```ruby
gem.add_dependency "docile", "~> 1.1"
```

Since the dependency practices semantic versioning, shouldn't upper-bound version be all the way to the indication of a breaking change, which would be `2.0`? So therefore it should be enough to say `"~> 1.1"`, which is equivalent to `">= 1.1.0", "< 2.0"`, right?

For reference: http://guides.rubygems.org/patterns/#declaring-dependencies

Reply:
yup it is. I just wasn't sure if docile followed semver (couldn't see it in the README) - so this was fine to me. If you say it does, then it's fine by me :) With relatively common dependencies such as simplecov I tend to be more conservative because more than a couple of times have new gem releases broken something for me.